R/check.R

```diff
@@ -173,60 +184,65 @@ coerce_dt <- function(
     }
   }
 
-  if ((length(required_cols) > 0)) {     # if we have required columns ...
-    if (!is.character(required_cols)) {  # ... check they are check-able
-      stop("`required_cols` must be a character vector")
+  if ((length(required_cols) > 0)) { # if we have required columns ...
+    if (!is.character(required_cols)) { # ... check they are check-able
+      rlang::abort("`required_cols` must be a character vector")
```

Comment:
Seems reasonable to add test that complains when required_cols isn't a character vector, to satisfy code coverage bot.